BackofficeBundle/EventSubscriber/DataTransformer/ArrayToStringTransformer.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\EventSubscriber\DataTransformer;
+
+/**
+ * Class ArrayToStringTransformer
+ */
+class ArrayToStringTransformer implements ValueTransformerInterface
+{
+    /**
+     * @param array $data
+     *
+     * @return string
+     */
+    public function transform($data)
+    {
+        $output = $data;
+        if (is_array($data)) {
+            $output = "<ul>";
+            foreach ($data as $item) {
+                $output .= "<li>" . $this->transform($item) . "</li>";
+            }
+            $output .= "</ul>";
+        }
```

Comment:
This data transformer is used to pregenerate the string value used to display complex types